docs/tutorials/01_introductory/viz_solar_system.py

```diff
@@ -80,6 +80,15 @@ def init_planet(filename):
 # of this tutorial, planet sizes and positions will not be completely
 # accurate.
 
+r_mercury = 7
+r_venus = 9
+r_earth = 11
+r_mars = 13
+r_jupiter = 16
+r_saturn = 19
+r_uranus = 22
+r_neptune = 25
```

Comment:
this feature can be set up at the same time as the texture. You could replace the texture list with a dict.